exams/models.py

```diff
@@ -35,3 +38,61 @@ class ExamProfile(Model):
 
     def __str__(self):
         return 'Exam Profile "{0}" with status "{1}"'.format(self.id, self.status)
+
+
+class ExamAuthorization(TimestampedModel):
+    """
+    Tracks state of an exam authorization
```

Comment:
Nit: what is an exam authorization? (I might just need to read some docs, but a better overview would be nice)